src/ck-ldb/LBManager.C

```diff
@@ -922,66 +920,71 @@ int LBRegisterObjUserData(int size)
   return CkpvAccess(lbobjdatalayout).claim(size);
 }
 
-
-class client {
-  friend class LocalBarrier;
-  void* data;
-  LDResumeFn fn;
+class LBClient {
```

Comment:
In the future will this just become a generic AtSync client? I get that it's named LBClient now because AtSync barriers are inside of the LB framework. But technically it's for more than just LB.

Reply:
It's "LB" for `LocalBarrier`, but I agree that it's way too ambiguous and confusingly named. Long term I want to remove AtSync from the load balancing stuff into its own individual class/file.

To answer your question, yes, it should become a generic AtSync client. However, this should basically only be used internally (as it is right now), so I don't think that will change this class much.

Reply:
Oh....wow I never would have guessed.

Reply:
I believe this was simply renaming existing class client -> LBClient and similarly existing class receiver-> LBreceiver to make them since they were used only in LB code. The renaming should probably have gone in a separate commit, since it mostly moving and renaming old code from LBDatabase.C to LBManager.C since it seemed to be functionality related to LBManager.

Reply:
Created #2779 for this.